doc/compile.md

```diff
@@ -0,0 +1,26 @@
+## Compile from Source Code
+
+To compile from source code, you'll need to have some prerequisite developer tooling
+installed on your machine.
+
+| Dependency                                          | Description                                                                                                                                                                          |
+| --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [golang](https://golang.org/doc/install)            | LiT's backend web server is written in Go. The minimum version supported is Go v1.13.                                                                                                |
+| [protoc](https://grpc.io/docs/protoc-installation/) | Required to compile LND & Loop gRPC proto files at build time                                                                                                                        |
+| [nodejs](https://nodejs.org/en/download/)           | LiT's frontend is written in TypeScript and built on top of the React JS web framework. To bundle the assets into Javascript & CSS compatible with web browsers, NodeJS is required. |
+| [yarn](https://classic.yarnpkg.com/en/docs/install) | a popular package manager for NodeJS application dependencies                                                                                                                        |
```

Comment:
nit: Capitalize sentence and add full stop, same for `protoc`.

Reply:
Fixed